CMSgov/mint-app
# [EASI-2970] Group Filtered Views

# EASI-2970

## Changes and Description

- Modified existing ReadOnly components to have a `isViewingFilteredView` view variant
- Add `BodyContent` filter view copmonent
- Payment sectino for ICP is TODO as that role is not implemented as of yet



## How to test this change

1. Login and go to the Read Only pages
2. Go through each filter to ensure the correct data is showing up (note: some questions are MIA because the task had not been filled out yet) 
3. Ensure questions in view matches the [spreadsheet](https://docs.google.com/spreadsheets/d/1HtXBREHtP5hEMFqcGvNDn_HuhyYMy9NX0tCvfWnRJd8/edit#gid=0)




## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
